I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 474, in script call
    call Nuria_finished_chapt1
  File "game/script.rpy", line 474, in script call
    call Nuria_finished_chapt1
  File "game/eduardo_days.rpy", line 77, in script
    if rachel_bus == 1:
  File "game/eduardo_days.rpy", line 77, in <module>
    if rachel_bus == 1:
NameError: name 'rachel_bus' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 476, in script call
    call atributes_nuria
  File "game/script.rpy", line 474, in script call
    call Nuria_finished_chapt1
  File "game/script.rpy", line 474, in script call
    call Nuria_finished_chapt1
  File "game/eduardo_days.rpy", line 77, in script
    if rachel_bus == 1:
  File "renpy/ast.py", line 1898, in execute
    if renpy.python.py_eval(condition):
  File "renpy/python.py", line 2276, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "renpy/python.py", line 2269, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/eduardo_days.rpy", line 77, in <module>
    if rachel_bus == 1:
NameError: name 'rachel_bus' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
A mess 1.0
Fri Jun  3 16:05:19 2022
